bpf/lib/nodeport.h

```diff
@@ -1729,6 +1739,15 @@ static __always_inline int nodeport_lb4(struct __ctx_buff *ctx,
 		if (!lb4_src_range_ok(svc, ip4->saddr))
 			return DROP_NOT_IN_SRC_RANGE;
 
+#if defined(ENABLE_L7_LB)
+		if (lb4_svc_is_l7loadbalancer(svc) && svc->l7_lb_proxy_port > 0) {
+			send_trace_notify(ctx, TRACE_TO_PROXY, src_identity, 0,
+					  bpf_ntohs((__u16)svc->l7_lb_proxy_port), 0,
+					  TRACE_REASON_POLICY, monitor);
+			return ctx_redirect_to_proxy_hairpin_ipv4(ctx,
```

Comment:
Good guestion, and this may require further scrutiny, as I arrived to the use of "hairpin" experimentally; The datapath did not work without the "hairpin" and works with it...